tests/tck/features/expression/FunctionCall.feature

```diff
@@ -40,3 +40,73 @@ Feature: Function Call Expression
     Then the result should be, in any order:
       | dt                           |
       | '2019-03-02T22:00:30.000000' |
+
+  Scenario: concat
+    When executing query:
+      """
+      GO FROM "Tim Duncan" over like YIELD concat(like._src, $^.player.age, $$.player.name, like.likeness) AS A
+      """
+    Then the result should be, in any order:
+      | A                             |
+      | "Tim Duncan42Manu Ginobili95" |
+      | "Tim Duncan42Tony Parker95"   |
+    When executing query:
+      """
+      GO FROM "Tim Duncan" over like YIELD concat(like._src, $^.player.age, null, like.likeness) AS A
+      """
+    Then the result should be, in any order:
+      | A    |
+      | NULL |
+      | NULL |
+    When executing query:
+      """
+      match (a:player)-[b:serve]-(c:team{name: "Lakers"}) where a.age > 45   return  concat(a.name,c.name)
```

Comment:
please format the query prettily, such as:

```
MATCH ...
WHERE ...
RETURN ...
```

Reply:
done